book/index.md

```diff
@@ -28,17 +28,16 @@ Those who understand trees in depth are more likely to identify tree
 shapes in their own data, unlocking superior solutions to their
 problems. Those who count Big-O analysis as a tool in their toolkits not
 only have a better vocabulary for comparing alternative strategies, they
-are also more cognizant overall of the trade offs we implicitly make
-every day. Those who see recursion everywhere see recursion everywhere
+are also more generally cognizant of the trade-offs we implicitly make
+every day. Those who see recursion everywhere see recursion everywhere...
 :)
```

Comment:
Now that I see this again, I think you should probably toss the smiley; the ellipsis is appropriately creepy, I think.